web/templates/stream_sidebar_row.hbs

```diff
@@ -10,10 +10,13 @@
 
             <a href="{{url}}" title="{{name}}" class="stream-name">{{name}}</a>
 
-            <span class="unread_mention_info"></span>
-            <span class="unread_count"></span>
-            <span class="masked_unread_count"></span>
+            <div class="stream-markers-and-controls">
+                <span class="unread_mention_info"></span>
+                <span class="unread_count"></span>
+                <span class="masked_unread_count"></span>
+            </div>
+
+            <span class="sidebar-menu-icon stream-sidebar-menu-icon hidden-for-spectators"><i class="zulip-icon zulip-icon-more-vertical" aria-hidden="true"></i></span>
```

Comment:
I think moving this inside `selectable_sidebar_block` just ends up moving inside this rule:

```
.selectable_sidebar_block {                                                                             
    cursor: pointer;                                                                                    
}                                                                                                       
```

Which is probably OK. Though maybe we should just get rid of that class on this sidebar, since it seems 95% about the right sidebar.

Reply:
Yeah. Once the grid stuff is complete, I plan to do a pass on things like misused ARIA attributes, and a class audit like this would be good to do, too. Adding this specific class to my todo list on that front.